apps/pace/app-icon.js

```diff
@@ -0,0 +1 @@
+require("heatshrink").decompress(atob("mEwwkBiIA/AH8QRYoX/AAcAZqwXuMIQYCiAcOO456OCwwACF5orEDghzOE4oZCLxw+GDAKsXeSIqEGBKKGBwIRFGBCfIC7p5RC7oGHC8RxFC453JAw4Xda5AIEC5IAPO6AXmO5BuHC67OIUA4aERyIXEIxAAJiAuFC5gTEcgpDNBwoWCIxp4CCwp1OCIYAEOiDFNDBwVQAH4AvA="))
```

Comment:
This draws black on black for me in `dtlaunch` (Desktop Launcher) when using Dark BW system theme.

Reply:
Ah yes, thanks! Sorted